hs_core/views/__init__.py

```diff
@@ -540,6 +541,11 @@ def copy_resource(request, shortkey, *args, **kwargs):
     return HttpResponseRedirect(new_resource.get_absolute_url())
 
 
+@api_view(['POST'])
+def copy_resource_public(request, pk):
```

Comment:
Can we make this function to return id of the copied resource and not the resource url?